src/format.rs

```diff
@@ -100,3 +127,242 @@ impl fmt::Display for BinaryBytes {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    const NANO: Duration = Duration::from_nanos(1);
+    const MILLI: Duration = Duration::from_millis(1);
+    const SECOND: Duration = Duration::from_secs(1);
+    const MINUTE: Duration = Duration::from_secs(60);
+    const HOUR: Duration = Duration::from_secs(60 * 60);
+    const DAY: Duration = Duration::from_secs(24 * 60 * 60);
+    const WEEK: Duration = Duration::from_secs(7 * 24 * 60 * 60);
+    const YEAR: Duration = Duration::from_secs(365 * 24 * 60 * 60);
```

Comment:
Why not pull these out of `tests` and use them in the `fmt::Display` impl as well, rather than redefining them every time?